modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -90,13 +93,33 @@ EOF
 +
 [source,terminal]
 ----
-oc get clusterserviceversion -n openshift-nmstate \
- -o custom-columns=Name:.metadata.name,Phase:.status.phase
+oc get pod -n openshift-nmstate
```

Comment:
@qiowang721 does this work?

Reply:
@JoeAldinger sorry maybe i did not describe clearly...
We need to keep the original Verification section as it is, the Succeeded csv means the operator is installed.
For the original step4, it should be a step after Verification, and we can use `oc get pod` or `oc get deployment` `oc get daemonset` to verify it.